src/test/java/jep/test/TestPyJType.java

```diff
@@ -0,0 +1,27 @@
+package jep.test;
+
+public class TestPyJType {
+
+    public static interface Root {
```

Comment:
Could it be worth adding some `default` methods here, to make sure that doesn't break stuff (now or in the future)?